app/src/main/java/com/sdk/getid/model/app/network/RequestUtilsOptions.kt

```diff
@@ -16,6 +16,6 @@ object RequestUtilsOptions {
     fun getRequestHeadersWithApiKey() = arrayListOf<Pair<String, String>>().apply {
         add(Pair(ACCEPT, APPLICATION_JSON))
         add(Pair(CONTENT_TYPE, APPLICATION_JSON))
-        add(Pair(API_KEY, AppSetupState.GET_ID_API_KEY))
+//        add(Pair(API_KEY, AppSetupState.GET_ID_API_KEY))
```

Comment:
The reason why we have that commented out is because the `ServiceParams.API_KEY` is no longer publicly available in the SDK, so it can not be imported and used. If it wasn't commented it would be breaking the build. 

@joelcassel